novawallet/Common/Extension/Storage/CDTransactionHistoryItem+CoreDataDecodable.swift

```diff
@@ -46,6 +50,14 @@ extension CDTransactionItem: CoreDataCodable {
         } else {
             txIndex = nil
         }
+        if let swapContainer = try? container.nestedContainer(keyedBy: SwapHistoryData.CodingKeys.self, forKey: .swap) {
+            let newSwap = CDTransactionSwapItem(context: context)
```

Comment:
We need to test this. I believe better first check whether current entity already has swap relation